Dockerfile

```diff
@@ -1,22 +1,20 @@
-FROM golang:1.18
-
-RUN mkdir -p /root/github.com/suborbital/subo
+FROM golang:1.18 AS builder
 WORKDIR /root/github.com/suborbital/subo
 
-# dependencies first
 COPY go.* ./
 RUN go mod download
 
-# then everything else
 COPY subo ./subo
 COPY builder ./builder
 COPY deployer ./deployer
 COPY packager ./packager
 COPY publisher ./publisher
 COPY project ./project
 COPY scn ./scn
-
 COPY *.go ./
 COPY Makefile .
-
 RUN make subo/docker-bin
+
+FROM debian:bullseye
+COPY --from=builder /go/bin/subo /go/bin/subo
+COPY --from=builder /go/bin/subo /usr/local/bin/subo
```

Comment:
Should we also put it on the path? We could link too.

Ideally we wanna clean up all these Dockerfiles and not using linking or rely on `$PATH`.